src/Components/dashboard/route/Skeletons/SkeletonElement.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Skeleton from'react-loading-skeleton';
+import {
+    Box,
+    Grid,
+    ListItemButton,
+    ListItemText,
+    Typography,
+  } from "@mui/material";
+<<<<<<< HEAD
+<<<<<<< HEAD
+import { Button } from "@mui/material";
+import './Skeleton.css';
+ 
+const SkeletonElement = ({cards})  => {  
+    return Array(cards).fill(0).map((item,i)=> (
+        <div className="card-skeleton" key={i}>          
+           <Grid container minHeight={30} spacing={1} >
+                <Grid item container minHeight={20} sx={{ width: "95%", margin: { xs: "auto", sm: "auto" }, display: "flex" }}>
+                     <Grid item sx={{ margin: { xs: 1, sm: 1 } }} xs={12} md={6.75} border={"1px solid #364878"}>
+                            <Box sx={{ mb: 1, px: 2}}>
+                                <Box my={0.5} sx={{ textAlign: 'center' }}>                          
+                                    <Skeleton count = {3}/>                                
+                                </Box>                                
+                          </Box>
+                </Grid>
+                 
+                  <Grid container item sx={{ margin: { xs: 1, sm: 1 } }} xs={12} md={2} >
+                    <Grid item xs={12} sx={{ mt: { xs: 1.5 } }} sm={4} md={12} >
+                      <Skeleton />
+                    </Grid>
+                    
+                  </Grid>
+                </Grid>
+              </Grid>
+=======
+import './Skeleton.css';
+ 
+const SkeletonElement = ({cards})  => {  
+    return Array(cards).fill(0).map((item)=> (
+        <div className="card-skeleton">          
+            <div className="main-col">
+                <Skeleton count = {2}/>
+            </div>
+>>>>>>> a7a64daa... (ft-routes-crud)
+=======
```

Comment:
Same here, you have to resolve these conflict in this file.

Reply:
thanks Abraham, Look like I missed this ones, I have resolved them